### PR TITLE
A better README

### LINKED ISSUE #10: Write a better readme
Document meeting times, places and where cancellations are communicated.

### PR DESCRIPTION
**Pull request by celaus**
Fixes #10, I hope 🤔 For the team members, please let me know who's missing, this is just a guess.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+The Rust events team meets every other week on [Telegram](https://t.me/joinchat/EkKINhHCgZ9llzvPidOssA) at 4 pm UTC, where we discuss any [open issues](https://github.com/rust-community/events/issues). We are happy to answer any questions, so feel free to comment or even resolve any issues in this repository 😁.  
+
+Alternatively, you can also email us at [community@rust-lang.org](mailto:community@rust-lang.org).
+
+## Team
+
+- [@badboy](https://github.com/badboy)
+- [@skade](https://github.com/skade)
+- [@flaki](https://github.com/flaki)
+- [@celaus](https://github.com/celaus)
```
Comment (skade) [inline README.md]: Missing: @mattgathu, otherwise great :)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
-Feel free to have a look at our [open issues](https://github.com/rust-community/events/issues).
+## Material
 
-If you want to contact us, please send an email to [community@rust-lang.org](mailto:community@rust-lang.org).
+- **Meetups:** Open an issue using [this template](https://github.com/rust-community/events-team/blob/master/ISSUE_TEMPLATE/new_meetup.md) ❤️
+
+
+# 🚀 Helping out
+
+The Rust events team meets every other week on [Telegram](https://t.me/joinchat/EkKINhHCgZ9llzvPidOssA) at 4 pm UTC, where we discuss any [open issues](https://github.com/rust-community/events/issues). We are happy to answer any questions, so feel free to comment or even resolve any issues in this repository 😁.
```
Comment (badboy) [inline README.md]: We're now at 5pm UTC (same as general community team meeting)

File: README.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+# 🗓 Rust Events team
 
-This is the repository for all planning of the Rust event team. We give hints, help and contacts for running events.
+This is the repository for any planning of the [Rust events team](https://github.com/rust-community/team#%EF%B8%8F-events). If you are running an event we would love to offer some tips and contacts along with our experience to make it as smooth as possible.
 
-Feel free to have a look at our [open issues](https://github.com/rust-community/events/issues).
+## Material
 
-If you want to contact us, please send an email to [community@rust-lang.org](mailto:community@rust-lang.org).
+- **Meetups:** Open an issue using [this template](https://github.com/rust-community/events-team/blob/master/ISSUE_TEMPLATE/new_meetup.md) ❤️
```
Comment (badboy) [inline README.md]: We can use this link here to directly open a new issue using the template:
https://github.com/rust-community/events-team/issues/new?template=new_meetup.md

**Comment by celaus** [comment]
@badboy @skade  Updated!

**Comment by celaus** [comment]
@badboy @skade updates!

**Pull request merged**